lib/Hutch.Rackit/Models/TaskApi/AvailabilityQuery.cs

```diff
@@ -2,6 +2,7 @@
 
 namespace Hutch.Rackit.Models.TaskApi;
 
+// TODO: Should models be shared by RACKit and the query library? and therefore in a lib of their own?
```

Comment:
I also thought about this - my instinct is yes, given it avoids repeating and reduces mistake.
Given we'll be putting the query lib in this repo also, I don't think there's much unhelpful overhead either?

Reply:
My alternative, to avoid proliferating packages, is that since these are API models, and the RACKit contains the API client, they continue to live in RACKit since it's where they logically belong, and the query lib then depends on RACKit (even though it doesn't use some of its functionality such as the actual Api Client)

Reply:
Moved this to #51 as it doesn't block the PR but might produce further work <3